### ISSUE TITLE
User bug report: Multiplayer drops frequently

### ISSUE THREAD

**Issue by toddmedema** [ExpeditionRPG/expedition#647]
"We had some issues (first game) where several of the devices would drop midgame, and have to restart the app. But would come back and resync. We were all on the same WiFi etc, 4 Android devices and 1 Apple iPhone. "

**Comment by toddmedema** [comment]
Related: "Could not stop combat timers. 2 player multiplayer. We stopped first combat, but had to press repeatedly to do so. One on iPhone and one android. Second combat we could not stop. We were in the same place so just played on one phone. You might want to offer a bypass if this remains a challenge."

Related: "There is definitely something happening to cause the sync between multiple devices to fail. I had another group last night of four and we couldn't even get the game started. When everyone would sign into the room it was signing people in 2 times, so our group of 4 had 8 people in the lobby. We eventually just used a tablet for all 4 of us and that worked. Also the signing people in multiple times happened on both the app and through the url you gave me."
